test/openAPI/features/support/data_list.js

```diff
@@ -1,74 +1,95 @@
-const pactum = require('pactum');
-const { When, Then, Given, After, Before } = require('@cucumber/cucumber');
-const { header, localhost } = require('./helpers/helpers');
+const { spec } = require('pactum');
+const chai = require('chai');
+const { When, Then, Given, Before } = require('@cucumber/cucumber');
+const {
+  header,
+  localhost,
+  dataListResponseSchema,
+  defaultExpectedResponseTime,
+  contentTypeHeader, dataListReadEndpoint
+} = require('./helpers/helpers');
+chai.use(require('chai-json-schema'));
 
-let searchedName;
 let specDataList;
 
-const baseUrl = `${localhost}data/registry1/version1`;
+const baseUrl = localhost + dataListReadEndpoint;
```

Comment:
Please add below endpointTag
`const endpointTag = { tags: `@endpoint=/${dataListReadEndpoint}` };`
and pass it to the Before and After func